packages/devtools_shared/test/server/devtools_extensions_api_test.dart

```diff
@@ -146,14 +146,14 @@ void main() {
 
   group(ExtensionsApi.apiExtensionEnabledState, () {
     late File optionsFile;
-    late final optionsFileUriString = p.join(
+    late final optionsFileUriString = p.posix.join(
```

Comment:
Any places where we're using `path.join` on URIs, we must use forward slashes (`path.posix` does this). Probably we should be using the URI class instead, but this works and avoids going in and out of `Uri` instances when the API is otherwise strings.